discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -348,9 +349,16 @@ def collect_entities_to_fetch(update_task, entity_manager_txs, metadata):
                             raw_address.lower()
                         )
                     else:
-                        logger.error(
-                            "tasks | entity_manager.py | Missing address in metadata required for add developer app tx"
-                        )
+                        try:
+                            entities_to_fetch[EntityType.DEVELOPER_APP].add(
```

Comment:
do we still need raw_address above? if this recovered address should match?

Reply:
yeah we'd need it for the delete case since we are only requiring the app signature payload for the create action. 
whereas we're keeping the delete action metadata the same as before (only requiring the address)

my rationale here is that once the user proves they own the developer app address ("delegate wallet") then they don't have to do it again

Reply:
oooh gotcha